spacialgaze-plugins/economy.js

```diff
@@ -114,6 +115,15 @@ function rankLadder(title, type, array, prop, group) {
 	return ladderTitle + tableTop + tableRows + tableBottom;
 }
 
+function economyStat() {
+	let users = Db.currency.keys(), hasOne, total = 0;
+	for (let i = 0; i < users.length; i++) {
+		if (Economy.readMoney(users[i]) === 1) hasOne++;
+		total += Economy.readMoney(users[i]);
```

Comment:
Incase of undefined or another weird non-number value, make this `total += Economy.readMoney(users[i]) || 0;`